package.json

```diff
@@ -24,7 +24,7 @@
     "coverage": "node $(npm bin)/istanbul cover -x \"**/third_party/**\" _mocha -- $(find */test -name '*-test.js') --timeout 10000 --reporter progress",
     "coveralls": "npm run coverage && cat ./coverage/lcov.info | coveralls",
     "start": "node ./lighthouse-cli/index.js",
```

Comment:
do we need to have pre-compiled templates for `global installs` ? currently we are ignoring those.
 Do you think, it makes sense to add them under `develop` section ?